src/main/java/hudson/plugins/jira/Updater.java

```diff
@@ -347,11 +348,11 @@ protected void appendAffectedFilesToDescription(Entry change, StringBuilder desc
 
     protected void appendChangeTimestampToDescription(StringBuilder description, JiraSite site, long timestamp) {
         DateFormat df = null;
-        if (!Strings.isNullOrEmpty(site.getDateTimePattern())) {
+        if (!StringUtils.isBlank(site.getDateTimePattern())) {
```

Comment:
nit: invert the statement while you are here - positive conditions are easier to understand

Reply:
good point